OptionSelector/OptionsTableViewController.m

```diff
@@ -0,0 +1,110 @@
+//
+//  OptionsTableViewController.m
+//  OptionSelector
+//
+//  Created by Kaisha Jones on 8/13/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OptionsTableViewController.h"
+
+
+@interface OptionsTableViewController ()
+
+@end
+
+@implementation OptionsTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+}
+
+
+#pragma mark - Table View data source 
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    // Return the number of sections.
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    // Return the number of rows in the section.
+    return self.category.options.count;
```

Comment:
The problem is that self.category.options is a string, not an array so count doesn't work.  If you use self.category.name.count this will work.  You want to get the count of each element we're displaying.

Since self.category is a CQCategory, the 'name' property is the array you want to display.  You can see the properties if you look in CQCategory.h.